code/game/machinery/portable_turret.dm

```diff
@@ -977,6 +977,7 @@ var/list/turret_icons
 
 /atom/movable/porta_turret_cover

```

Comment:
what the actual hell.

Reply:
Yeah, that made me do a double-take too.